pages/getting-started/understand-odata-in-6-steps.html

```diff
@@ -1358,4 +1358,7 @@ <h4>Contribute to "Understanding OData in 6 steps"</h4>
             $('.btn-default').focus();
        });
     })
+
+    // make it possible to tab to response content inside popups
+    $(".modal-body").attr("tabindex", 0);
```

Comment:
Add tab index to the pre since it has the content thus allowing keyboard users and screen readers to be able to navigate.

Reply:
Also $(document).ready :)